### DISCUSSION TITLE
_printChanges Documentation Not Found

### DISCUSSION THREAD

**Discussion by brzzdev** [pointfreeco/swift-composable-architecture#2141] [Q&A]
### Description

When in the [Docs](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/) and trying to navigate to the _printChanges documentation in the sidebar it results in "The page you’re looking for can’t be found." and from there a refresh results in "404 File not found"

Tested in both Firefox and Chrome, also in private windows.

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-composable-architecture/issues) or [discussion](https://github.com/pointfreeco/swift-composable-architecture/discussions).

### Expected behavior

Document page is available

### Actual behavior

Document page is not available

### Steps to reproduce

Navigate to _printChanges in sidebar

**Comment by mbrandonw** [comment]
Hi @PaulDoesDev, this appears to be a bug in Apple's current tools. By default, underscored methods are not exposed in documentation, but you can use [`@_documentation(visibility:public)`](https://github.com/pointfreeco/swift-composable-architecture/blob/4aab2176b5441ea3d37e6c38fe6bd7bd1fc5d0ed/Sources/ComposableArchitecture/Reducer/Reducers/DebugReducer.swift#L12) to expose it. That allows us to link to the docs, but unfortunately the page doesn't seem to exist. Hopefully this will be fixed in Apple's tools soon.

Since this is not an issue with the library I am going to convert it to a discussion. Maybe someone else has an idea of how to work around this problem.

**Comment by brzzdev** [comment]
Ah, I didn't realise that that particular documentation was something you get from Apple. Makes sense why I see it in so many packages 😅

Thanks for the prompt response
